README.md

```diff
@@ -183,6 +183,7 @@ You can use `CPM_SOURCE_CACHE` on GitHub Actions workflows [cache](https://githu
 If set, CPM will forward all calls to `CPMFindPackage` as `CPMAddPackage`.
 This is useful to create reproducible builds or to determine if the source parameters have all been set correctly.
 This can also be set as an environmental variable.
+This can be controlled on a per project basis with the `CPM_<dependency name>_DOWNLOAD` variable.
```

Comment:
also I'm not sure about the name `CPM_<dep>_DOWNLOAD`. 

`CPM_Boost_DOWNLOAD` doesn't sound like proper English.

How about `CPM_DOWNLOAD_<dep>` 

"Hey, CPM, download this package!"

Reply:
That makes alot of sense as it also better matches `CPM_DOWNLOAD_ALL` (Hopefully we never encounter a package named ALL ).